src/Startup.cs

```diff
@@ -33,7 +37,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             // Serve pn.json
             app.Run(async context =>
             {
-                if (context.Request.Path == $"/{Program.PnJson}" &&
+                if (context.Request.Method == "GET" && context.Request.Path == $"{Program.PnJson}" &&

```

Comment:
Does this work without forward slash as prefix? $"{Program.PnJson}"
If so change: $"{Program.PnJson}" -> Program.PnJson

Reply:
good catch. for the default value i had to add a /.